tools/codegen/gen_functionalization_type.py

```diff
@@ -29,6 +32,11 @@
 #     Generates view_copy() composite kernels for all view_copy operators.
```

Comment:
This comment needs a nice, long update

Reply:
yeah... it's actually up-to-date again now that I removed all of the AddBackView kernels